website/content/docs/connect/config-entries/file-system-certificate.mdx

```diff
@@ -1,21 +1,23 @@
 ---
 layout: docs
-page_title: File System Certificate Configuration Reference
+page_title: File system certificate - Configuration entry reference
 description: Learn how to configure a file system certificate bound to an API Gateway on VMs.
 ---
 
-# File system certificate configuration reference
+# File system certificate configuration entry reference
```

Comment:
Out of scope, but I noticed that the different pages for config entries reference have different titles and capitalization. 

Two examples:
https://consul-5r7uaucy3-hashicorp.vercel.app/consul/docs/connect/config-entries/http-route
https://consul-5r7uaucy3-hashicorp.vercel.app/consul/docs/connect/config-entries/terminating-gateway

we might want to align the titles.